src/libraries/sfx.proj

```diff
@@ -11,10 +11,12 @@
     <ProjectReference Include="sfx-ref.proj">
       <OutputItemType Condition="'$(RefOnly)' == 'true'">SharedFrameworkAssembly</OutputItemType>
     </ProjectReference>
-    <!-- Support building only the reference assemblies. -->
-    <ProjectReference Include="sfx-src.proj"
-                      OutputItemType="SharedFrameworkAssembly"
-                      Condition="'$(RefOnly)' != 'true'" />
+  </ItemGroup>
+
+  <!-- Support building only the reference assemblies. -->
+  <ItemGroup Condition="'$(RefOnly)' != 'true'">
+    <ProjectReference Include="sfx-src.proj" OutputItemType="SharedFrameworkAssembly" />
+    <ProjectReference Include="sfx-gen.proj" />
```

Comment:
Can you double check that we still include the analyzers in the ref-pack after this change?  Also the ASP.NET transport package.

Reply:
Just verified this locally by building from a clean state and the analyzers are correctly placed into the targeting packs' and the packages' analyzers/dotnet/(cs)/ folder.

Reply:
I opened https://github.com/dotnet/runtime/issues/69233 though as the LibraryImportGenerator might only work for C# projects and is currently not marked as lanague specific.